firmware/user/modes/mode_tunernome.c

```diff
@@ -455,16 +455,48 @@ static void ICACHE_FLASH_ATTR tunernomeUpdate(void* arg __attribute__((unused)))
             {
                 case GUITAR_TUNER:
                 {
+                    // Mode name
                     plotText((OLED_WIDTH - textWidth(theWordGuitar, IBM_VGA_8)) / 2,
                              (OLED_HEIGHT - FONT_HEIGHT_IBMVGA8) / 2,
                              theWordGuitar, IBM_VGA_8, WHITE);
+
+                    // Note names of strings, arranged to match LED positions
+                    for(int i = 0; i < NUM_GUITAR_STRINGS / 2; i++)
+                    {
+                        plotText((OLED_WIDTH - textWidth(theWordGuitar, IBM_VGA_8)) / 2 - textWidth("G4 ", IBM_VGA_8),
+                                 (OLED_HEIGHT - FONT_HEIGHT_IBMVGA8) / 2 + (FONT_HEIGHT_IBMVGA8 + 5) * (1 - i),
+                                 guitarNoteNames[i], IBM_VGA_8, WHITE);
+                    }
+                    for(int i = NUM_GUITAR_STRINGS / 2; i < NUM_GUITAR_STRINGS; i++)
+                    {
+                        plotText((OLED_WIDTH + textWidth(theWordGuitar, IBM_VGA_8)) / 2 + charWidth(' ', font_IbmVga8),
```

Comment:
pls just use `textWidth(" ", IBM_VGA_8)` instead because I'm a monster

Reply:
![endMyPain](https://user-images.githubusercontent.com/666399/96536045-311c5480-1261-11eb-8e71-c666562c9849.png)

Reply:
gone, like it never happened